de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/InventoryAllocationDestination.java

```diff
@@ -417,8 +423,9 @@ private I_M_InventoryLine getCreateInventoryLine(
 		final ISnapshotProducer<I_M_HU> currentSnapshotProducer = huSnapshotProducerByInventoryId.get(inventoryLine.getM_Inventory_ID());
 		Check.errorIf(currentSnapshotProducer == null, "Missing SnapshotProducer for M_Inventory_ID={}", inventoryLine.getM_Inventory_ID());
 
+		// add each top level HU to be added to the snapshot of the inventory
 		currentSnapshotProducer.addModel(topLevelHU);
-		currentSnapshotProducer.createSnapshots();
+
```

Comment:
hi, the currentSnapshotProducer.createSnapshots(); you moved was in the method naed `createHuSnapshot()`.
I arranged it that way in this commit: 5d8d73b4c1165b7d8f85b048641fd2dcc2988421
* please check out the problem i tried to address, and if it's still solve with this change (i guess "yes")
* please figure aout a more fitting name for this method (you can also or consider inlining it..whatever makes the code more readable)